lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -1310,9 +1307,9 @@ TypeSP DWARFASTParserClang::ParseSubroutine(const DWARFDIE &die,
       }
     }
   }
-  return std::make_shared<Type>(
-      die.GetID(), dwarf, attrs.name, llvm::None, nullptr, LLDB_INVALID_UID,
-      Type::eEncodingIsUID, &attrs.decl, clang_type, Type::ResolveState::Full);
+  return dwarf->MakeType(die.GetID(), attrs.name, llvm::None, nullptr,
```

Comment:
Are these all swift-lldb specific patches, or are these changes also upstream?

Reply:
The first two commits are upstream, the last one is lldb-swift specific.

Reply:
Sorry — I keep forgetting that you *can* look at individual commits' contributions in github's web interface.